packages/socket-stream-client/server.js

```diff
@@ -12,3 +12,7 @@ setMinimumBrowserVersions({
   safari: 7,
   electron: [0, 20],
 }, module.id);
+
+if (process.env.DISABLE_SOCKJS) {
```

Comment:
This needs to be properly documented. Please add this to new env var to documentation.

Reply:
Done.